.golangci.yml

```diff
@@ -53,6 +53,7 @@ issues:
     # TODO: Needs manual update
     - text: "package github.com/golang/protobuf/proto is deprecated: .*"
       linters: [staticcheck]
+  new-from-rev: dd12c4c34cd3d90327cc6882e83df58902301318
```

Comment:
I think if that commit is not the last one when you merge, then if any new lint warning appeared between that commit and your merge it will make the linter fail

Reply:
Right after the `7.48.0` freeze, I'll update the `new-from-rev` with the last commit on `main` after the freeze.
I'll ask Kacper to unfreeze my PR and merge it right away, and then tell the Agent teams to rebase/merge `main` in their branch.